### PR TITLE
readme: Add archive note

### PR DESCRIPTION
**Pull request by kevpar**
Signed-off-by: Kevin Parsons <kevpar@microsoft.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# Archived
+
+This project has been archived. Its contents have been merged into [Microsoft/hcsshim](https://github.com/microsoft/hcsshim) where development will continue. Some code organization has changed as part of the merge, please see [the merge PR](https://github.com/microsoft/hcsshim/pull/973) for details on where things have moved. Any issues regarding opengcs should be opened on the hcsshim repo.
```
Comment (kevpar) [inline README.md:3]: @dcantah if you have details on the code organization changes I can include here, please let me know.

Comment (dcantah) [reply README.md:3]:
```suggestion
This project has been archived. Its contents have been merged into [Microsoft/hcsshim](https://github.com/microsoft/hcsshim) where development will continue. Some code organization has changed as part of the merge, please see below for details on where things have moved. Any issues regarding opengcs should be opened on the hcsshim repo.

Changes:
* Everything in /internal has been moved to [github.com/Microsoft/hcsshim/internal/guest](https://github.com/microsoft/hcsshim/tree/master/internal/guest) besides a few redundant packages that had equivalents in hcsshim. These include /internal/debug, /internal/oc and /internal/log. These are now located at github.com/Microsoft/hcsshim/internal/debug, github.com/Microsoft/hcsshim/internal/oc, and github.com/Microsoft/hcsshim/internal/log.
* /service/libs/commonutils has moved to github.com/Microsoft/hcsshim/internal/guest.
* Any non Go code lives at the root of github.com/Microsoft/hcsshim now, just as it did in this repository. This includes /vsockexec, /init, and /hack.
* The main gcs and gcstools binaries are now built and located at github.com/Microsoft/hcsshim/cmd/gcs and  github.com/Microsoft/hcsshim/cmd/gcstools. 
* Makefile lives top level at github.com/Microsoft/hcsshim/Makefile and same commands will work to build everything.
```

Comment (kevpar) [reply README.md]: Thanks @dcantah, looks good!

File: README.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
+
+This project has been archived. Its contents have been merged into [Microsoft/hcsshim](https://github.com/microsoft/hcsshim) where development will continue. Some code organization has changed as part of the merge, please see below for details on where things have moved. Any issues regarding opengcs should be opened on the hcsshim repo.
+
+Changes:
+* Everything in /internal has been moved to [github.com/Microsoft/hcsshim/internal/guest](https://github.com/microsoft/hcsshim/tree/master/internal/guest) besides a few redundant packages that had equivalents in hcsshim. These include /internal/debug, /internal/oc and /internal/log. These are now located at github.com/Microsoft/hcsshim/internal/debug, github.com/Microsoft/hcsshim/internal/oc, and github.com/Microsoft/hcsshim/internal/log.
+* /service/libs/commonutils has moved to github.com/Microsoft/hcsshim/internal/guest.
+* Any non Go code lives at the root of github.com/Microsoft/hcsshim now, just as it did in this repository. This includes /vsockexec, /init, and /hack.
+* The main gcs and gcstools binaries are now built and located at github.com/Microsoft/hcsshim/cmd/gcs and  github.com/Microsoft/hcsshim/cmd/gcstools.
```
Comment (dcantah) [inline README.md:9]: Nit: There's an extra space I accidentally left in between github.com/Microsoft/hcsshim/cmd/gcs and  github.com/Microsoft/hcsshim/cmd/gcstools

**Review by dcantah — APPROVED** [review]
lgtm!

File: README.md (hunk 2 above)
Comment (kevpar) [reply README.md]: Fixed

**Review by anmaxvl — APPROVED** [review]

**Comment by kevpar** [comment]
CI is taking forever, we don't upkeep the code in here anymore, and this PR only affects the readme, so I am going to merge without CI completion.

**Pull request merged**